patches/react-server-dom-webpack@18.3.0-next-1308e49a6-20230330.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/cjs/react-server-dom-webpack-client.browser.development.js b/cjs/react-server-dom-webpack-client.browser.development.js
+index bc2bf7106366df38e04d705a950b4170ae01571f..f6df80dde6a5ddcc8356dddf98f6797fb70f2748 100644
+--- a/cjs/react-server-dom-webpack-client.browser.development.js
++++ b/cjs/react-server-dom-webpack-client.browser.development.js
+@@ -73,7 +73,7 @@ function preloadModule(metadata) {
+     var entry = chunkCache.get(chunkId);
+ 
+     if (entry === undefined) {
+-      var thenable = __webpack_chunk_load__(chunkId);
++      var thenable = __webpack_require__.e(chunkId);
```

Comment:
do we not have runtime variable shorthand methods avaliable yet like __webpack_chunk_load__ or __resourceQuery?

Reply:
Webpack will replace `__webpack_chunk_load__` to `__webpack_require__.e`.

Rspack now support `__webpack_chunk_load__` too. The patch isn't necessary now.